project_4.html

```diff
@@ -0,0 +1,524 @@
+			<h3 class="mb-3 fw-light" style="display: inline;">LED Blinker</h3>
+			<ul class="mb-5">
+				<li>You must build a blinking LED circuit.</li>
+				<li>The LED in the circuit must blink once per second.</li>
+				<li>The LED must be brightly lit when turned ON.</li>
+				<li>The LED must be completely dark when turned OFF.</li>
+				<li>The circuit must be built on a breadboard.</li>
+			</ul>
+			<h3 class="mb-3 fw-light" style="display: inline;">LED Dimmer</h3>
+			<ul class="mb-5">
+				<li>You must build a circuit with a dimmable LED.</li>
+				<li>The user must be able to adjust the LED’s brightness from 0% to 100% when turning the potentiometer dial.</li>
+				<li>When turning the potentiometer clockwise, the LED should brighten.</li>
+				<li>When turning the potentiometer counter-clockwise, the LED should dim.</li>
+				<li>The circuit must be built on a breadboard.</li>
+			</ul>
+			<h3 class="mb-3 fw-light" style="display: inline;">RGB LED Dimmer</h3>
+			<ul class="mb-5">
+				<li>You must build a circuit with a dimmable RGB LED.</li>
+				<li>The user must be able to adjust the RGB LED to any color when turning the three potentiometer dials.</li>
+				<li>Each potentiometer should control the light intensity of one of the three LED colors (red, green, and blue).</li>
+				<li>When turning a potentiometer clockwise, its respective LED color should brighten.</li>
+				<li>When turning a potentiometer counter-clockwise,  its respective LED color should dim.</li>
+				<li>The circuit must be built on a breadboard.</li>
+			</ul>
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-parts">Parts</h2>
+
+			<!-- ======= Part Table ======= -->
+			<table class="table pj-part-table mb-5">
+				<thead>
+					<tr>
+						<th scope="col">Part Name</th>
+						<th scope="col">Qty</th>
+					</tr>
+				</thead>
+				<tbody>
+					<tr>
+						<td>Jumper Wire</td>
+						<td>X?</td>
+					</tr>
+					<tr>
+						<td>Breadboard</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>Potentiometer, 10kΩ</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>LED, 3.2V</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>LED, RGB, Common Cathode</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>
+							<a class="pj-part"
+								data-bs-content="<img src='./assets/images/parts/ops_part_arduino_nano.png' height='150px' width='auto'>"
+								data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0">
+								Arduino Nano
+							</a>
+						</td>
+						<td>
+							1
+						</td>
+					</tr>
+					<tr>
+						<td>Resistor, 130Ω</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>Resistor, 430Ω</td>
+						<td>2</td>
+					</tr>
+					<tr>
+						<td>Resistor, 470Ω</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>Mini USB Cable</td>
+						<td>1</td>
+					</tr>
+				</tbody>
+			</table>
+			<!-- End Part Table -->
+
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-schematics">Schematics</h2>
```

Comment:
Please reduce the max-width of the images for Schematics A and B to be more consistent with the part sizing of Schematic C